.github/workflows/verify-app-build.yml

```diff
@@ -0,0 +1,255 @@
+#        path: 'beeware-.github'
+
+    # TODO:PR: replace with commented block above before merging
+    - name: Checkout rmartin16/.github-beeware
+      uses: actions/checkout@v3.3.0
+      with:
+        repository: 'rmartin16/.github-beeware'
+        path: 'beeware-.github'
+        ref: 'shared-ci'
+
+    - name: Package name
+      id: package
+      run: echo "name=$(basename '${{ inputs.repository }}')" >> ${GITHUB_OUTPUT}
+
+    - name: Determine Cache Directories
+      id: dirs
+      run: |
+        echo "cookiecutters-data-dir=~/.cookiecutters" >> ${GITHUB_OUTPUT}
+
+        if [[ "${{ startsWith(inputs.runner-os, 'ubuntu') }}" == "true" ]]; then
+          echo "briefcase-data-dir=~/.cache/briefcase" >> ${GITHUB_OUTPUT}
+          echo "pip-cache-dir=~/.cache/pip" >> ${GITHUB_OUTPUT}
+        
+        elif [[ "${{ startsWith(inputs.runner-os, 'macos') }}" == "true" ]]; then
+          echo "briefcase-data-dir=~/Library/Caches/org.beeware.briefcase" >> ${GITHUB_OUTPUT}
+          echo "pip-cache-dir=~/Library/Caches/pip" >> ${GITHUB_OUTPUT}
+          echo "docker-cache-dir=~/Library/Containers/com.docker.docker/Data/vms/0/" >> ${GITHUB_OUTPUT}
+        
+        elif [[ "${{ startsWith(inputs.runner-os, 'windows') }}" == "true" ]]; then
+          echo "briefcase-data-dir=~/AppData/Local/BeeWare/briefcase/Cache" >> ${GITHUB_OUTPUT}
+          echo "pip-cache-dir=~/AppData/Local/pip/Cache" >> ${GITHUB_OUTPUT}
+          echo "docker-cache-dir=C:/ProgramData/DockerDesktop" >> ${GITHUB_OUTPUT}
+        fi
+
+    - name: Cache Briefcase tools
+      uses: actions/cache@v3.2.3
+      with:
+        key: briefcase-${{ runner.os }}-${{ inputs.repository }}-${{ inputs.framework }}-${{ inputs.target-platform }}-${{ inputs.target-format }}
+        path: |
+          ${{ steps.dirs.outputs.cookiecutters-data-dir }}
+          ${{ steps.dirs.outputs.briefcase-data-dir }}
+          ${{ steps.dirs.outputs.pip-cache-dir }}
+          ${{ steps.dirs.outputs.docker-cache-dir }}
+
+    - name: Set up Python
+      if: inputs.python-source == 'github'
+      uses: actions/setup-python@v4.5.0
+      with:
+        python-version: ${{ inputs.python-version }}
+
+    - name: Install Windows Store Python
+      if: inputs.python-source == 'winstore'
+      uses: ./beeware-.github/.github/actions/install-win-store-python
+      with:
+        python-version: ${{ inputs.python-version }}
+
+    - name: Get packages
+      # Briefcase will build and package itself in a previous step in its CI.
+      if: endsWith(inputs.repository, 'briefcase')
+      uses: actions/download-artifact@v3.0.2
+      with:
+        name: packages-${{ steps.package.outputs.name}}
+        path: dist
+
+    - name: Install Briefcase artifact
+      if: endsWith(inputs.repository, 'briefcase')
+      run: python -m pip install dist/briefcase-*.whl
+      
+    - name: Install Briefcase
+      if: endsWith(inputs.repository, 'briefcase') != true
```

Comment:
Is this not just `!endWith...`?

Reply:
definitely is....my brain didn't want to believe `!` was supported or something...

Updated.

Reply:
interestingly....GitHub allows `endsWith(inputs.repository, 'briefcase') != true` but requires the brackets for `${{ !endsWith(inputs.repository, 'briefcase') != true }}`....hmm...